test/libyul/StackShufflingTest.cpp

```diff
@@ -137,6 +141,10 @@ StackShufflingTest::StackShufflingTest(std::string const& _filename):
 
 TestCase::TestResult StackShufflingTest::run(std::ostream& _stream, std::string const& _linePrefix, bool _formatted)
 {
+	auto const& dialect = EVMDialect::strictAssemblyForEVMObjects(
+		solidity::test::CommonOptions::get().evmVersion(),
+		solidity::test::CommonOptions::get().eofVersion()
+	);
```

Comment:
We're using this pattern often enough that we should probably define it as `CommonOptions::dialect()`.

Reply:
-> separate PR

Reply:
PR #15618